libr/debug/p/native/linux/linux_debug.c

```diff
@@ -227,6 +227,10 @@ int linux_step(RDebug *dbg) {
 }
 
 bool linux_set_options(RDebug *dbg, int pid) {
+	if (dbg->options_set) {
+		return true;
+	}
```

Comment:
This change will prevent updating options for a given PID after initial attachment.